salsa_dancing_molecules/lennardjonesparse.py

```diff
@@ -12,12 +12,15 @@ def parse_lj_params(atom_letters):
     """
     with open('LennardJones612_UniversalShifted.params') as param_file:
```

Comment:
The L-J path is not independent of the current working directory. This parameter file should be moved into the module and the script has to figure out where the file is to open or the parameter file needs to be provided on the command line.